dev/dev-overrides.env

```diff
@@ -0,0 +1,2 @@
+PORT=3000
```

Comment:
Couple of requests:
1. Can we place this file in the root of the project?
1. Can we gitignore this file?

Reply:
ok, sure

Reply:
are we changing the location of all the .env files to root directory or only this file?

Reply:
Done